daml-lf/spec/transaction.rst

```diff
@@ -434,27 +417,21 @@ As of version 10, these fields are included:
 * `message VersionedValue`_ return_value
 * `message KeyWithMaintainers`_ key_with_maintainers
 
-``contract_id_struct`` is required. Its structure is defined by `the value
-specification`_, version 3.
+``contract_id_struct`` is required. 
 
-``children`` may be empty; all other fields are required, and required
-to be non-empty.
+``children`` and ``key_with_maintainers`` may be empty; all other
+fields are required, and required to be non-empty.
 
-``template_id``'s structure is defined by `the value specification`_;
-the version of that specification to use when consuming it is the
-``version`` field of ``chosen_value``.
+``template_id``'s structure is defined by `the value specification`_.
```

Comment:
same question what happened to the version field? I thought we still have that in 10 or am I off by one?

Reply:
In original design the field ``template_id`` was interpreted according the version of the ``chosen_value``.
In practice this feature has never been used, and was drop when redesigning the versioning scheme. 
no need to speak of the``version`` field of ``chosen_value` here.